routes/index.js

```diff
@@ -25,6 +25,18 @@ router.get('/:day(-?\\d*)?', counter.countVisitors, function (req, res, next) {
         });
 });
 
+router.get('/cityfood/:day(-?\\d*)?', counter.countVisitors, function (req, res, next) {
```

Comment:
Mir wäre rein optisch die Route `/city` lieber (food steckt eh schon in subdomain).

Reply:
ist ein gutes Argument :+1: